### ISSUE TITLE
Creating after copying ISOs

### ISSUE THREAD

**Issue by rrking858** [pr1ntf/iohyve#202]
I was unable to create a new guest. Some probing showed that the variable "pool" was:

zroot
zroot
zroot
zroot

Further probing showed that the line that generated the variable pool (ioh-guest, line 166) was getting multiple values because I had copied in some of the ISOs that I had lying on my system for future use under iohyve. Appending "| tail -n1" to the chain appears to have fixed the problem.

**Comment by pr1ntf** [comment]
Thank you for the detailed report. When you copied over the ISO files, did you do it manually, or did you use the `cpiso` function?

**Comment by rrking858** [comment]
I used the cpiso command.
I tried to install Ubuntu last night and had issues. I will be looking into those today. I have already done this by hand, so if I can find the command in your scripts, I should be able to diagnose the problem.
-- Randy King 

```
On Sunday, July 31, 2016 12:11 AM, pr1ntf <notifications@github.com> wrote:
```

 Thank you for the detailed report. When you copied over the ISO files, did you do it manually, or did you use the cpiso function?—

**Comment by pr1ntf** [comment]
Did you install via ports or from the GitHub repo? I believe that ISO issue with no guests installed has been fixed.

**Comment by rrking858** [comment]
Installed via ports (pkg install iohyve). I will try installing from GitHub before trying to install Ubuntu. I didn't get to it today.--RRK 

```
On Sunday, July 31, 2016 8:57 PM, pr1ntf <notifications@github.com> wrote:
```

 Did you install via ports or from the GitHub repo? I believe that ISO issue with no guests installed has been fixed. —

**Comment by pr1ntf** [comment]
No rush. If you have nothing of value in your `iohyve` dataset, I would ask you to nuke it and all of it's children before you try again. If not, no worries. I just want to solve your ISO issue. If you are having issues with Ubuntu not related to the ISO issue, I'd ask if you could open a separate issue for proper tracking.

**Comment by rrking858** [comment]
FYI, the Ubuntu problem was a lack of memory. Default value in iohyve is 256M, but the Ubuntu installation needs more (512M?). --RRK 

```
On Tuesday, November 8, 2016 8:02 AM, pr1ntf <notifications@github.com> wrote:
```

 Closed #202.—

**Comment by pr1ntf** [comment]
Thanks for the info!
